docs/modules/getting-started/pages/enterprise-overview.adoc

```diff
@@ -0,0 +1,13 @@
+= Overview
+:description: This section provides a series of quick tutorials for setting up Hazelcast clusters equipped with the following Enterprise features.
+
+{description}
+
```

Comment:
NOTE: To complete these tutorials, you'll need Docker. See xref:getting-started:get-started-enterprise.adoc#using-the-enterprise-docker-image[Using the Enterprise Docker Image].

In this series, you will learn how to do the following: